contracts/test/v0.8/dev/FunctionsOracle.test.ts

```diff
@@ -82,12 +82,12 @@ describe('FunctionsOracle', () => {
     await registry.setAuthorizedSenders([oracle.address])
 
     await registry.setConfig(
-      1_000_000,
-      86_400,
-      21_000 + 5_000 + 2_100 + 20_000 + 2 * 2_100 - 15_000 + 7_315,
-      ethers.BigNumber.from('5000000000000000'),
-      500_000,
-      300,
+      1_000_000, // maxGasLimit
+      86_400, // stalenessSeconds
+      39_173, // gasAfterPaymentCalculation
```

Comment:
was that changed on purpose from 44615? Also why not use a nice round value for tests?

Reply:
Yes changed on purpose - I ran the exact gas numbers, this is currently a consistent value unless something changes in the code after payment.